public/js/test/mochitest/examples/README.md

```diff
@@ -0,0 +1,5 @@
+### Test Examples
+
+##### Pages
+* **doc_script-switching-01** - includes two scripts that reference each other. The second function has a debugger.
+* **doc-scripts** - includes three sources, a long source and two sources that reference each other.
```

Comment:
Ok, we can try it. I agree that we should avoid duplicating test cases. In the old tests, most of them use just a few of the examples. The two that we have right now will probably cover a lot of them. Of course, there are plenty of edge cases that will need new examples for specific things. Maybe in the readme we could make a not about the "common" test cases that you should probably use unless you know what you're doing.